docs/guide/different_vehicles.md

```diff
@@ -0,0 +1,333 @@
+
+The contents of the world file uses the same syntax as your normal model files. See the examples for adding primitive objects. For models, you will need to include the models inside this directory (and add that folder onto your path in the next step).
+#### Building the model (`setup.bash`)
+
+Similar to setup for the custom model, we need to define a setup for the custom world. For worlds, this usually just involves ensuring that the world and asscoiated models are on the gazebo paths:
+
+```bash
+SRC_PATH="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Add flightarena.world to the Gazebo resource path
+export GAZEBO_RESOURCE_PATH=$SRC_PATH/worlds:${GAZEBO_RESOURCE_PATH}
+echo "RESOURCE_PATH: $GAZEBO_RESOURCE_PATH"
+
+# Add the 'grass_box' model to the Gazebo model path
+export GAZEBO_MODEL_PATH=$SRC_PATH/models:${GAZEBO_MODEL_PATH}
+echo "GAZEBO_MODEL_PATH: $GAZEBO_MODEL_PATH"
+
+# Make the default media files work
+cp -r /usr/share/gazebo-11/media $GZWEB_WS/http/client/assets/
+(cd /root/gzweb/http/client/assets/media/materials/textures \
+    && for f in *jpg; do convert $f ${f%.*}.png; done)
+```
+
+The final line is to ensure that all of the textures are compatible with the Gazebo web interface. 
+
+### Building and Running the models
+
+Now that all of the models and worlds have been created, we then need to either bundle it up, or inject it into the container so it can be included. 
+
+There are two methods. Both of which are applicable depending on eventual usecase. 
+
+1. Docker-Compose: The models/worlds can be mounted into the simulation container at runtime
+2. Building a new Docker Container: If you know that you want to build upon the models or worlds into multiple differing applications, it's probably best to package it up and use as a dependency for the child projects. 
+
+#### Docker-Compose
+
+As mentioned earlier, as long as the folders have a `setup.bash` script inside, it is compatible with automation within `/ros.env.d`. Therefore you can mount your folders into the container as volumes within the docker-compose file:
+
+```yaml
+simhost:
+    image: uobflightlabstarling/starling-sim-iris:${STARLING_RELEASE:-latest}
+    volumes:
+      - ./flightarena:/ros.env.d/02_flightarena
+      - ./systems/models/gimbal_small_2d:/ros.env.d/03_gimbal_small_2d
+      - ./systems/models/gimbal_tripod:/ros.env.d/04_gimbal_tripod
+      - ./system.launch.xml:launch/system.launch.xml
+    env:
+      - GIMBAL_NAMESPACE=gimbal_1 
+    command: ["ros2", "launch", "launch/system.launch.xml"]
+    ports:
+      - "8080:8080"
+```
+
+Where the syntax is `<local file>:<container file>`. Also note that inside `ros.env.d` we name the folders `XX_<name>` where `XX` is a two digit number describing the order of script loading from 0 being first to 99 being last. This is useful for dependencies.
```

Comment:
Should probably be `00` for the first to avoid ordering surprises...

Reply:
Changed!